Dockerfile.special-resource-operator-build

```diff
@@ -14,4 +14,9 @@ RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/i
 RUN GO111MODULE=on go install golang.stackrox.io/kube-linter/cmd/kube-linter@latest
 RUN curl -L --retry 5 "https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv4.3.0/kustomize_v4.3.0_linux_amd64.tar.gz" | \
     tar -zx -C /usr/bin
-RUN go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.0
+RUN go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.1
+RUN go install github.com/golang/mock/mockgen@v1.5.0
+RUN go install github.com/onsi/ginkgo/ginkgo@v1.16.4
+# RUN go install golang.org/x/tools/cmd/goimports@v0.1.5
+RUN go get github.com/vektra/mockery/.../@v1.1.2
```

Comment:
Why do we need this?

Reply:
i think that one of the goals should be to run build and unittest locally, and not using Dockerfile.ubi. So we need those package in the skipper. Currently you can compile using make manager command, but later, we can add/check running unittest also

Reply:
My question was specifically about mockery - I don't think it's used in the PR?

Reply:
aaa-a, sorry, did not understood from the comment.  2 things:
1) preparation for the next commit
2) controller-gen needs to access the mock package in the vendor directory when it passes over the generated mock file. So i added it to the skipper, and then moved it to the vendor

Reply:
fixed